.github/workflows/tests.yml

```diff
@@ -206,3 +206,31 @@ jobs:
         v=${GITHUB_REF:10}
         docker tag brew "docker.pkg.github.com/homebrew/brew/brew:$v"
         docker push "docker.pkg.github.com/homebrew/brew/brew:$v"
+
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tag: [ubuntu18, ubuntu20]
```

Comment:
The tag above is `brew:$v`, here it will be something like `ubuntu18:$v`. Not sure this is really consistent. But I also do not see a real alternative. Could we at least have 18.04 in the name; as there is also a 18.10 release.

Reply:
I prefer the name `ubuntu18` over `ubuntu18.04`. We have no plans to ship a non-LTS release of Ubuntu, and if we did, that image could be called `ubuntu18.10`.

Reply:
That's just my preference. I can be swayed if the popular opinion is `ubuntu18.04`.

Reply:
I've made the change to use `ubuntu18.04` and `ubuntu20.04`.